src/smockit/types.ts

```diff
@@ -1,5 +1,17 @@
 /* Imports: External */
-import { Contract } from 'ethers'
+import { Contract, ContractFactory, ContractInterface } from 'ethers'
+
+export type SmockSpec =
+  | ContractInterface
+  | Contract
+  | ContractFactory
+  | string
+  | any
```

Comment:
"any" here really needs to be the structure of the standard JSON abi. Not sure if that type is easily accessible anywhere?